uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/PublicHolidayLeaveRequestTest.php

```diff
@@ -92,9 +94,11 @@ public function testUpdateAllInTheFutureForContract() {
    * created after the contract gets saved.
    */
   public function testItUpdateAllInTheFutureWhenTheContractDetailsAreCreated() {
+    WorkPatternFabricator::fabricateWithA40HourWorkWeek(['is_default' => 1]);
+    //All the days selected for the public holidays are working days for the 40hr work week
     $datePublicHoliday1 = new DateTime('yesterday');
```

Comment:
If you run this test on a monday, then yesterday will be sunday and the test will fail